### PR TITLE
Transit UI

### PR DESCRIPTION
**Pull request by Antichess**
add station markers, along with full read-json compatibility

### CODE REVIEW

File: src/output.txt [hunk 1]
```diff
@@ -0,0 +1,36 @@
+43.6646250,-79.3996975,Robarts Library Cafeteria
```
Comment (geotrieu) [inline src/output.txt:1]: What is this file, and why is it in `src`?

File: transit-ui/src/components/dict.json [hunk 2]
```diff
@@ -0,0 +1,57 @@
+{
```
Comment (geotrieu) [inline transit-ui/src/components/dict.json]: Lets not put this in `src/components`. Maybe, lets make a new folder inside `transit-ui`, maybe `data`

File: transit-ui/src/components/hardcode_json.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import json
```
Comment (geotrieu) [inline transit-ui/src/components/hardcode_json.py]: Is this a test script to generate `dict.json`? Lets make a new folder under `transit-ui` called `test` and put this there instead.

**Review by geotrieu — CHANGES_REQUESTED** [review]

File: src/output.txt (hunk 1 above)
Comment (Antichess) [reply src/output.txt:1]: accidentally made this file, will put in gitignore

File: transit-ui/src/components/hardcode_json.py (hunk 3 above)
Comment (Antichess) [reply transit-ui/src/components/hardcode_json.py]: yes, this is a test script. i moved the output into a new folder named `data`.

Comment (Antichess) [reply transit-ui/src/components/hardcode_json.py]: moved `hardcode_json.py` into `src/test`

**Review by geotrieu — APPROVED** [review]

**Pull request merged**